subgraph/schema.graphql

```diff
@@ -26,3 +27,16 @@ type Key @entity {
   createdAtBlock: BigInt!
   cancelled: Boolean
 }
+
+type LockDayData @entity {
+  id: ID!
+  lockDeployed: BigInt!
+  keysSold: BigInt!
```

Comment:
You are initializing this but not actually incrementing it

Reply:
fixed